tests/test_user_service.py

```diff
@@ -72,3 +72,43 @@ async def test_register_user_record_exists(
     assert exc_info.value.status_code == 500
     assert exc_info.value.detail == "User already exists"
     mock_user_repository.save.assert_called_once_with(mock_user)
+
+@pytest.fixture
+def mock_user():
+    # Return a mock user object (replace with your actual mock user)
+    return {
+        "id": 1,
+        "username": "Ann",
+        "email": "Ann.aj@example.com",
+        "first_name": "Ann",
+        "last_name": "aj",
+        "phone_number": "94872",
+        "role": "user",
+        "status": "active",
+    }
```

Comment:
There is already a mock defined for this in line 22, Please reuse that and remove this.

https://github.com/Null-By-Design/user-service/blob/e8131429947338f0770b6e9e98c9c56de6eb461c/tests/test_user_service.py#L23